code/modules/core_implant/cruciform/rituals/priest.dm

```diff
@@ -216,6 +216,30 @@
 		if(target.wearer && target.wearer.stat != DEAD)
 			return target
 
+/datum/ritual/cruciform/priest/announcement
+	name = "Address the Flock"
+	phrase = "Cum autem magna conquisitio fieret, surgens Petrus dixit ad eos." //"After much discussion, Peter got up and addressed them"
+	desc = "This litany acts as a mass Sending, addressing all disciples."
+	category = "Episcopal"
+	power = 30
+
+/datum/ritual/cruciform/priest/announcement/perform(mob/living/carbon/human/user, obj/item/implant/core_implant/cruciform/C)
+	var/text = input(user, "What message will you speak to the Church? The message will be recieved telepathically.", "Sending a message") as text|null
```

Comment:
```suggestion
	var/text = input(user, "What message will you speak to the Church?", "Sending a message") as text|null
```

Reply:
Yeah, that's just what Sending uses. I can probably do a quick reflavor on that